Misc/NEWS.d/next/Library/2019-08-29-18-48-48.bpo-37587.N7TGTC.rst

```diff
@@ -0,0 +1,2 @@
+Use ``_PyUnicodeWriter`` instead of list of unicode chunks in
```

Comment:
`_PyUnicodeWriter` is a deep implementation detail. `_json.scanstring` is also an implementation detail. The end user should not be aware of them. Just describe the effect of the optimization. It is worth to add also a What's New entry.